### ISSUE TITLE
Test scan_id doesn't appear in the relationships.json. How should I build my test dataset?

### ISSUE THREAD

**Issue by yyliucch** [ShunChengWu/3DSSG#4]

**Comment by ShunChengWu** [comment]
The test scans should not be seen of a network in the training stage. Since our evaluation relies on the labeled ground truth mesh file, we use the validation set from 3RScan as our test set.

We split the training sequences into "training set" and "validation set". The "test set" we use here is actually the validation set. 

You can split the training sequences into whatever portion you want and save them into a [NAME].txt. Then pass it to our data generation script `--target_scan=/path/to/[NAME].txt`.

**Comment by glennliu** [comment]
Hi @ShunChengWu 

I tried using the "validataion set" mode to generate the "test set", in the data generation script. It still requires to load an groundtruth label file. Anything I missed?

Thanks
